src/material-experimental/mdc-slider/slider.ts

```diff
@@ -22,13 +52,218 @@ import {SliderAdapter} from './slider-adapter';
   selector: 'mat-slider',
   templateUrl: 'slider.html',
   styleUrls: ['slider.css'],
+  host: {
+    'class': 'mat-mdc-slider mdc-slider',
+    '[class.mdc-slider--range]': 'isRange',
+    '[class.mdc-slider--disabled]': 'disabled',
+    '[class.mdc-slider--discrete]': 'discrete',
+    '[class.mdc-slider--tick-marks]': 'showTickMarks',
+  },
+  exportAs: 'matSlider',
   changeDetection: ChangeDetectionStrategy.OnPush,
   encapsulation: ViewEncapsulation.None,
 })
-export class MatSlider {
+export class MatSlider implements AfterViewInit, OnDestroy {
+  /** The slider thumb(s). */
+  @ViewChildren('thumb') _thumbs: QueryList<ElementRef<HTMLElement>>;
 
-  /** Instance of the MDC slider foundation for this slider. */
+  /** The slider thumb knob(s) */
+  @ViewChildren('knob') _knobs: QueryList<ElementRef<HTMLElement>>;
+
+  /** The active section of the slider track. */
+  @ViewChild('trackActive') _trackActive: ElementRef<HTMLElement>;
+
+  /** The sliders hidden range input(s). */
+  @ContentChildren(MatSliderThumb, {descendants: false}) _inputs: QueryList<MatSliderThumb>;
+
+  /** Whether the slider is disabled. */
+  @Input()
+  get disabled(): boolean { return this._disabled; }
+  set disabled(v: boolean) {
+    this._disabled = coerceBooleanProperty(v);
+    if (this._initialized) {
```

Comment:
Why is the `_initialized` check needed? It looks like `_initialized` is also never being set

Reply:
Whoops, this is meant to be set to true `ngAfterViewInit`. I'll fix that.

It's needed because if we want to disable the slider after the component has been initialized we have to also inform the foundation with `this._foundation.setDisabled(v);`. And we can't just call it every time because if the foundation hasn't been initialized it will throw errors

Reply:
Ack, can you add a comment explaining that?

Also, adding unit tests as you go is the thing that typically catches oversights like this. I'd recommend doing future additions alongside with unit tests, otherwise they'll start to get hard to manage.